packages/taxdat/R/plot_WHOcomparison_table.R

```diff
@@ -80,9 +80,18 @@ plot_WHOcomparison_table <- function(config, cache, cholera_directory, observati
               ")"
             )
       })
-    who_annual_cases$modeled_ci <- modeled_obs_level_cases
-
+     who_annual_cases_ci <- cache[["sf_cases_resized"]] %>% sf::st_drop_geometry()
 
+    who_annual_cases_ci$modeled_ci <- modeled_obs_level_cases
+    who_annual_cases_ci_no_imputation <- who_annual_cases_ci %>% filter(!is.na(loctime))
```

Comment:
dplyr::filter -- I know we missed the explicit package name in some places, but we should try to do it as much as possible